### DISCUSSION TITLE
Remotely Viewing Logs

### DISCUSSION THREAD

**Discussion by k3vyn32** [ESPresense/ESPresense#333] [Q&A]
Reviewing the Terminal output via the Web Terminal is useful, but if you have many ESP's around the home, it would be useful to view the logs remotely. Either via a gui or command line, is this possible at this time?

**Comment by DTTerastar** [comment]
It would be nice.  BUT the main thing i'd want to see is the crash logs, and those aren't able to be sent AFAIK since, well it's crashed...
